tests/foreman/cli/test_activationkey.py

```diff
@@ -648,8 +642,8 @@ def test_positive_add_redhat_and_custom_products(self):
             u'organization-id': self.org['id'],
         })
         self.assertEqual(len(content), 2)
-        self.assertEqual(content[0]['name'], REPOSET['rhst7'])
-        self.assertEqual(content[1]['name'], repo['name'])
+        self.assertIn(REPOSET['rhst7'], [pc['name'] for pc in content])
+        self.assertIn(repo['name'], [pc['name'] for pc in content])
```

Comment:
Bonus. Was debugging this test and in my case the order was opposite (`content[0]['name']` was equal to `repo['name']`, `content[1]['name']` was equal to `REPOSET['rhst7']`).
However it won't fix the test on CI as it's failing because of race condition.